common/openapi.yml

```diff
@@ -23,126 +23,167 @@ paths:
       operationId: getVersion
       security: [ ]
       responses:
-        '200':
+        200:
           description: Version information
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/VersionProperties'
-        '404':
+        404:
           description: "Version not configured"
-        '500':
+        500:
           $ref: '#/components/responses/ServerError'
 
 
   # Specific terra scientific pipelines service queries will go here
-  /api/jobs/v1alpha1/{pipelineId}:
+
+  /api/pipelines/v1alpha1:
+    get:
+      summary: Return all available Pipelines
+      operationId: getPipelines
+      tags: [ pipelines ]
+      responses:
+        200:
+          description: Success
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/GetPipelinesResult'
+        403:
+          $ref: '#/components/responses/PermissionDenied'
+        404:
+          $ref: '#/components/responses/NotFound'
+        500:
+          $ref: '#/components/responses/ServerError'
+
+
+  /api/pipelines/v1alpha1/{pipelineId}:
     parameters:
       - $ref: '#/components/parameters/PipelineId'
     get:
-      summary: Retrieve all jobs requested by user
-      operationId: getJobs
-      tags: [ jobs ]
+      summary: Return info about the specified pipeline
+      operationId: getPipeline
+      tags: [ pipelines ]
       responses:
-        '200':
+        200:
           description: Success
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/GetJobsResponse'
-        '403':
+                $ref: '#/components/schemas/Pipeline'
+        403:
           $ref: '#/components/responses/PermissionDenied'
-        '404':
+        404:
           $ref: '#/components/responses/NotFound'
-        '500':
+        500:
           $ref: '#/components/responses/ServerError'
     post:
-      summary: Create new job request
+      summary: Create new pipeline job
```

Comment:
just to make sure im on the right track, eventually this will require a jobid to be passed in, in which case we will also need to add a 202 reponse code when we do?

Reply:
that's right - TSPS-136 covers requiring the jobid in the payload here. good catch about the 202 - that ought to be part of this PR, will add it.